src/swarm/neo/client/mixins/ClientCore.d

```diff
@@ -702,6 +702,22 @@ template ClientCore ( )
             static assert(false, "Invalid request type");
         }
     }
+
+    /***************************************************************************
+
+        Disables TCP socket output data buffering. Should be called before
+        adding a node.
+
+        Warning: This is meant to be used only in tests which perform sequential
+        requests. For a high data throughput rate it may impact performance so
+        do not use it in a production environment.
+
+    ***************************************************************************/
+
+    public void enableSocketNoDelay ( )
```

Comment:
Though this PR will be included in a patch release, this change is a new feature. It should have release notes.